Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -1092,20 +1109,18 @@ public virtual void Impact(Thing hitThing)
 
                     // Apply suppression around impact area
                     if (explodePos.y < SuppressionRadius)
-                        suppressThings.AddRange(GenRadial.RadialDistinctThingsAround(explodePos.ToIntVec3(), Map, SuppressionRadius + def.projectile.explosionRadius, true).OfType<Pawn>());
+                        suppressThings.AddRange(explodePos.ToIntVec3().PawnsInRange(Map, SuppressionRadius + def.projectile.explosionRadius));
                 }
-
-                if (explodingComp != null)
+                else if (explodingComp != null)
                 {
                     explodingComp.Explode(this, explodePos, Map, 1f, dir, ignoredThings);
 
                     if (explodePos.y < SuppressionRadius)
-                        suppressThings.AddRange(GenRadial
-                            .RadialDistinctThingsAround(explodePos.ToIntVec3(), Map, SuppressionRadius + (explodingComp.props as CompProperties_ExplosiveCE).explosiveRadius, true).OfType<Pawn>());
+                        suppressThings.AddRange(explodePos.ToIntVec3().PawnsInRange(Map, SuppressionRadius + (explodingComp.props as CompProperties_ExplosiveCE).explosiveRadius));
                 }
 
                 foreach (var thing in suppressThings)
-                    ApplySuppression(thing);
+                    ApplySuppression(thing, explosionSuppressionFactor);
```

Comment:
Just like `SuppressionRadius`, this should be `TitleCase`.  Not that CE is super consistent with that, so not a *huge* deal.